src/modules/add-and-remove.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable import/prefer-default-export */
+import { Task } from './to-to';
+import { input } from './html-element';
+// js for add and delete functionalities
+
+class functionalty {
+  static addNewTask() {
+    // js to create a new task
+    // js to setup local storage
+    if (localStorage.getItem('myTasks') === null) {
+      localStorage.setItem('myTasks', '[]');
+    }
+
+    const newTaskDescription = input.value;
+    const checked = false;
+    const taskslist = JSON.parse(localStorage.getItem('myTasks'));
+    const newTaskId = taskslist.length;
```

Comment:
- [ ] Here you have to modify the `newTaskId` for the new task because as required, the new ID should be as the new array length. So, if you add the 1st task, the ID should be 1, the 2nd - 2, the 3d - 3 and etc.
![image](https://user-images.githubusercontent.com/49064106/200077924-96d06801-9f26-4bac-9d36-6ad25ca11956.png)

Reply:
ok sir